package.json

```diff
@@ -79,24 +79,24 @@
 		"tree-sitter-css": "^0.19.0",
 		"tree-sitter-c-sharp": "^0.19.0",
 		"tree-sitter-cli": "^0.20.1",
-		"tree-sitter-clojure": "github:sogaiu/tree-sitter-clojure#master",
+		"tree-sitter-clojure": "github:sogaiu/tree-sitter-clojure#0.0.9",
 		"tree-sitter-cpp": "^0.19.0",
 		"tree-sitter-go": "^0.19.0",
-		"tree-sitter-haskell": "github:tree-sitter/tree-sitter-haskell#d6ccd2d9c40bdec29fee0027ef04fe5ff1ae4ceb",
+		"tree-sitter-haskell": "github:tree-sitter/tree-sitter-haskell#1b54c3f39436bbded6593ac9e0103f9115bbbd2e",
 		"tree-sitter-html": "^0.19.0",
 		"tree-sitter-java": "^0.19.1",
 		"tree-sitter-javascript": "^0.19.0",
 		"tree-sitter-json": "^0.19.0",
-		"tree-sitter-latex": "github:latex-lsp/tree-sitter-latex#1666e9780de42a31b1376d32fbe8332e8dd5850f",
+		"tree-sitter-latex": "github:latex-lsp/tree-sitter-latex#0.1.0",
 		"tree-sitter-markdown": "^0.7.1",
-		"tree-sitter-php": "github:tree-sitter/tree-sitter-php#0ce134234214427b6aeb2735e93a307881c6cd6f",
+		"tree-sitter-php": "github:tree-sitter/tree-sitter-php#b9b719ded9a1c4f3fe29848306cfbd6ed4544282",
 		"tree-sitter-python": "^0.19.0",
-		"tree-sitter-ruby": "github:tree-sitter/tree-sitter-ruby#1ebfdb288842dae5a9233e2509a135949023dd82",
+		"tree-sitter-ruby": "github:tree-sitter/tree-sitter-ruby#6334d6ab3d04a5672da695d3b155ca3301511f8d",
 		"tree-sitter-rust": "^0.19.0",
-		"tree-sitter-scala": "github:tree-sitter/tree-sitter-scala#master",
-		"tree-sitter-scss": "github:serenadeai/tree-sitter-scss#c478c6868648eff49eb04a4df90d703dc45b312a",
+		"tree-sitter-scala": "github:tree-sitter/tree-sitter-scala#8599058ef292e82203a1b23d10734dcbaf4d1b5c",
+		"tree-sitter-scss": "github:serenadeai/tree-sitter-scss#1.0.0",
 		"tree-sitter-sparql": "^0.1.0",
-		"tree-sitter-typescript": "github:tree-sitter/tree-sitter-typescript#master",
+		"tree-sitter-typescript": "^0.20.1",
```

Comment:
I thought we decided to pin to hashes?  cc/ @wenkokke

Reply:
Happy either way. 

Either way, based on the convo with `tree-sitter` team, since all of their repos are sub v1, we need to treat any upgrade as a very possible breaking change. I'm still a little hazy on the release and CI flows for this repo and how it relates to the test suite in the main cursorless repo. Maybe we can talk through it on discord.